ChopChop/Models/Ingredients/IngredientReference.swift

```diff
@@ -0,0 +1,24 @@
+/**
+ Represents some quantity of an ingredient.
+ */
+struct IngredientReference {
+    let name: String
+    private(set) var quantity: Quantity
+
+    init(name: String, quantity: Quantity) {
+        self.name = name
+        self.quantity = quantity
+    }
```

Comment:
Doesn't seem to have the same checks `Ingredient` has, which seems odd. If you create an `IngredientReference` with an empty name, it's not going to match any ingredient.

Reply:
I agree that it should have the same checks, I'll add it in